src/components/Layout/NavBar/NavMenu.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="nav-menu">
+    <div v-for="(menuItem, index) in menuData" :key="index" class="menu-item">
+      <router-link :to="menuItem.route">
```

Comment:
we should try to use method-based router navigation versus router-link elements when we get more complex and dynamic routes, router-links will not be able to provide as much control as easily than using a method to push the new route to the router via a click listener would provide.